public/views/JoinDatasets.vue

```diff
@@ -29,15 +29,28 @@
     </div>
 
     <div class="row flex-1 pb-3 h-100">
-      <variable-facets
-        class="col-12 col-md-3 d-flex h-50 pt-2"
-        enable-search
-        enable-type-change
-        enable-highlighting
-        :instance-name="instanceName"
-        :rows-per-page="numRowsPerPage"
-        :summaries="variableSummaries"
-      />
+      <div class="h-100">
```

Comment:
You need to add the class `col-md-3` to work on with the next `<div>` on screen smaller than 1000px or so.